synapse/rest/synapse/client/oidc/backchannel_logout_resource.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 The Matrix.org Foundation C.I.C.
```

Comment:
```suggestion
# Copyright 2022 The Matrix.org Foundation C.I.C.
```

Reply:
Oh. Well, I started working on this patch in 2021 :D